lib/.eslintrc.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  env: {
+    jest: true
```

Comment:
I'm not sure the whole folder will only run in Jest. My understanding is that some methods in the `helper-functions.js` file will be run in a plain `Node` environment while `Gulp` runs. If we say they run in `jest` we risk allowing globals that may not be available when they run some otherway (though we'll likely uncover those when we try to build the project anyway).

Reply:
Yeah, `file-helper.js` has some stuff that's used outside of `jest` as well.

Reply:
Good feedback, I'll narrow this down